src/template/submission_process.md

```diff
@@ -0,0 +1,6 @@
+# Submission Process
+
+To submit a notebook for publication in this gallery, completet the following steps:
++ Upload a .zip of your notebook and all associated files to the IN-CORE zenodo community [here](https://zenodo.org/communities/in-core/records?q=&l=list&p=1&s=10&sort=newest). *Note*, if you are using a Mac, please remove all `.DS_Store` and `__MACOSX/` files and directories using the following commands in the terminal `zip -d filename.zip \*/.DS_Store` and `zip -d filename.zip __MACOSX/\*` respectively. 
++ When submitting your notebook through zenodo, the following attributes are required: co-authors, <span style="color:red">etc. </span>
```

Comment:
also, please add a case of just a single notebook file

Reply:
when I upload, there were more items to consider like DOI. Would you be able to add little more details in this part?

Reply:
I updated this. I'd like y'all's thoughts about if there is anything else that should be included. 
I also combined the `submission_guidelines.md` and `submission_process.md` files to a single `submission.md` file.